docs/eventing/broker-trigger.md

```diff
@@ -13,29 +13,63 @@ A Broker represents an 'event mesh'. Events are sent to the Broker's ingress and
 are then sent to any subscribers that are interested in that event. Once inside
 a Broker, all metadata other than the CloudEvent is stripped away (e.g. unless
 set as a CloudEvent attribute, there is no concept of how this event entered the
-Broker).
+Broker). There can be different classes of `Broker`s providing different kinds
+of semantics around durability of events, performance, etc. The `Broker` that is
+part of the Knative Eventing repo is used for these examples, it uses Knative
+[Channels](./channels/) for delivering events. Simple example showing a `Broker`
+where the configuration is specified in a `ConfigMap` config-br-default-channel,
+which uses `InMemoryChannel`. You can read more details about
+[Channel Based Broker](./channel-based-broker.md)
 
 Example:
 
 ```yaml
-apiVersion: eventing.knative.dev/v1alpha1
+apiVersion: eventing.knative.dev/v1beta1
 kind: Broker
 metadata:
   name: default
 spec:
-  channelTemplateSpec:
-    apiVersion: messaging.knative.dev/v1alpha1
-    kind: InMemoryChannel
+  # Configuration specific to this broker.
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: config-br-default-channel
+    namespace: knative-eventing
+```
+
+More complex example, showing the same `Broker` as above
+but with failed events being delivered to Knative Service called `dlq-service`
+
+```yaml
+apiVersion: eventing.knative.dev/v1beta1
+kind: Broker
+metadata:
+  name: default
+spec:
+  # Configuration specific to this broker.
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: config-br-default-channel
+    namespace: knative-eventing
+  # Where to deliver Events that failed to be processed.
+  delivery:
+    deadLetterSink:
+      ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: dlq-service
 ```
 
 ## Trigger
 
 A Trigger represents a desire to subscribe to events from a specific Broker.
 
-Example:
+Simple example which will receive all the events from a given (`default`) broker and
+delivers them to Knative Serving service `Service`:
```

Comment:
typo nit: "deliver". Also, the name of the service is "my-service" not "Service"